npm-shrinkwrap.json

```diff
@@ -2,6 +2,18 @@
   "name": "etcher",
   "version": "1.0.0-beta.19",
   "dependencies": {
+    "ajv": {
```

Comment:
We were previously using the HTTP functionality from the Angular framework. `request` is the most stable and mature solution for NodeJS project, which I guess outweighs the hassle of having extra dependencies. Also consider that we'll make heavy use of this library when implementing the catalog, given that React doesn't ship with HTTP functionality :)